### PR TITLE
DOT updates

### PR DESCRIPTION
**Pull request by laboon**
This updates the DOT and FAQ pages to include the warning about transferring mainnet DOTs and information about DOT sales (text from Jack, with some very minor additions to clarify that this refers to mainnet and not testnet DOTs)

### CODE REVIEW

**Comment by laboon** [comment]
Ready for review @ansonla3 / @lsaether

File: docs/polkadot/learn/DOT.md [hunk 1]
```diff
@@ -34,12 +34,20 @@ DOTs will be used to facilitate the consensus mechanism that underpins Polkadot.
...
+
+Testnet dots are freely available now - see below for various ways to obtain them.
+
 ## Testnet DOTs
 
 DOTs are required to make transactions on the Polkadot network. Testnet DOTs do not have any value beside allowing you to experiment with the network.
 
 ### Getting testnet DOTs
 
-You can request DOTs from the [faucet](https://faucets.blockxlabs.com/) ask someone to send you DOTs on the [Polkadot Watercooler](https://riot.im/app/#/room/#polkadot-watercooler:matrix.org).
```
Comment (ansonla3) [inline docs/polkadot/learn/DOT.md:43]: and this one
https://faucet.polkadot.chat/ 
:P

**Review by ansonla3 — CHANGES_REQUESTED** [review]

File: docs/polkadot/learn/DOT.md (hunk 1 above)
Comment (lsaether) [reply docs/polkadot/learn/DOT.md:43]: we will switch to https://faucet.polkadot.network soon too

Comment (ansonla3) [reply docs/polkadot/learn/DOT.md:43]: later on, we can redirect those traffic to that?

Comment (laboon) [reply docs/polkadot/learn/DOT.md:43]: I'd say we switch over to  https://faucet.polkadot.network once it's live - until then we can leave it up as .chat.

**Review by ansonla3 — APPROVED** [review]

**Pull request merged**